python-project-template/.github/workflows/{% if mypy_type_checking != 'none' %}type-checking.yml{% endif %}.jinja

```diff
@@ -36,4 +36,4 @@ jobs:
 {% elif mypy_type_checking == 'strict' %}
       run: |
         mypy ./src ./tests --strict
-{% endif %}
+{% endif -%}
```

Comment:
I still don't have confidence in my understanding of the whitespace control. Wouldn't this remove the new line at the end of the file when the template is hydrated?

Reply:
Or, wait is that the point? Does this end up with two empty lines when the template is hydrated?

Reply:
Yes, that is the point. We want only one '\n' character at the end of the file when hydrated.
Do we have a test for this case?

Reply:
We don't have a test for this case. Until now, aside from the __init__.py file, the EOL hasn't been a big focus.